style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.card_memmory {
+  width: 150px;
+  height: 150px;
+  background-color: aquamarine;
+  margin: 15px 15px;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-child {
```

Comment:
I suggest u to rethink the name of this child since we're not all childrens will be driven by `row`